generate_compose/odcs_compose_generator.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+"""Initialize and use a compose generator using ODCS"""
+
+from pathlib import Path
+
+import click
+import yaml
+
+from .compose_generator import ComposeGenerator
+from .odcs_configurations_generator import ODCSConfigurationsGenerator
+from .odcs_fetcher import ODCSFetcher
+from .odcs_requester import ODCSRequester
+
+
+@click.command()
+@click.option(
+    "--compose-path",
+    help="Path to where the compose should be saved.",
+    type=click.Path(path_type=Path),
+    required=True,
+)
+@click.option(
+    "--container-yaml-path",
+    help="Path in which container yaml file is stored",
+    type=click.Path(path_type=Path),
+    required=True,
+)
+@click.option(
+    "--content-sets-yaml-path",
+    help="Path in which content_sets yaml file is stored",
+    type=click.Path(path_type=Path),
+    required=True,
+)
+def main(
+    compose_path: Path,
+    container_yaml_path: Path,
+    content_sets_yaml_path: Path,
+):
+    """
+    Get inputs from container and content_sets yamls and relay them to an ODCS
+    compose generator that will request a compose and store it in a TBD location.
+    """
+    container_data: dict = yaml.safe_load(container_yaml_path.read_text())
+    content_sets_data: dict = yaml.safe_load(content_sets_yaml_path.read_text())
+
+    compose_generator = ComposeGenerator(
+        configurations_generator=ODCSConfigurationsGenerator(
+            container_data=container_data,
+            content_sets_data=content_sets_data,
+        ),
+        requestor=ODCSRequester(),
+        fetcher=ODCSFetcher(compose_path=compose_path),
```

Comment:
If you want it to fit my change, I'll be renaming it to `compose_file_path`

Reply:
done